configs/mmdet/mmdet_regression_test.yaml

```diff
@@ -0,0 +1,175 @@
+globals:
+  codebase_name: mmdet
+  codebase_release: V2.22.0
+  codebase_dir: ../mmdetection
+  checkpoint_force_download: False
+  checkpoint_dir: ../mmdeploy_checkpoints
+  dataset_path:
+    COCO:
+  metric_tolerance:
+    box AP: 0.1
+    mask AP: 0.1
+    PQ: 0.1
+  default_deploy_config: &default_deploy_config
+    onnxruntime: &default_onnxruntime_config
+      performance_align: False
+      deploy_config:
+        static:
+          fp32: &default_onnxruntime_static_fp32 detection_onnxruntime_static.py
+        dynamic:
+          fp32: &default_onnxruntime_dynamic_fp32 detection_onnxruntime_dynamic.py
+    tensorrt: &default_tensorrt_config
+      performance_align: False
+      deploy_config:
+        static:
+          fp32: &default_tensorrt_static_fp32 detection_tensorrt_static-300x300.py
+          fp16: &default_tensorrt_static_fp16 detection_tensorrt-fp16_static-800x1344.py
+          int8: &default_tensorrt_static_int8 detection_tensorrt-int8_static-800x1344.py
+        dynamic:
+          fp32: &default_tensorrt_dynamic_fp32 detection_tensorrt_dynamic-320x320-1344x1344.py
+          fp16: &default_tensorrt_dynamic_fp16 detection_tensorrt-fp16_dynamic-320x320-1344x1344.py
+          int8: &default_tensorrt_dynamic_int8 detection_tensorrt-int8_dynamic-320x320-1344x1344.py
+    openvino:  &default_openvino_config
+      performance_align: False
+      deploy_config:
+        static:
+          fp32: &default_openvino_static_fp32
+        dynamic:
+          fp32: &default_openvino_dynamic_fp32
+    ncnn: &default_ncnn_config
+      performance_align: False
+      deploy_config:
+        static:
+          fp32: &default_ncnn_static_fp32
+        dynamic:
+          fp32: &default_ncnn_dynamic_fp32
+    pplnn: &default_pplnn_config
+      performance_align: False
+      deploy_config:
+        static:
+          fp32: &default_pplnn_static_fp32
+        dynamic:
+          fp32: &default_pplnn_dynamic_fp32
+    sdk: &default_sdk_config
+      performance_align: False
+      deploy_config:
+        static:
+          fp32: &default_sdk_static_fp32
+          fp16: &default_sdk_static_fp16
+          int8: &default_sdk_static_int8
+        dynamic:
+          fp32: &default_sdk_dynamic_fp32
```

Comment:
we assume sdk only have one config, say dynamic: fp32: configs/mmdet/detection/detection_sdk_dynamic.py for detection

Reply:
Done